pkg/controller/installation/installation_controller.go

```diff
@@ -40,7 +39,7 @@ import (
 )
 
 const (
-	deletionFinalizer                = "foregroundDeletion"
+	deletionFinalizer                = "kubernetes"
```

Comment:
`kubernetes` finalizer name is used by k8s itself, so IMHO we should use a custom finalizer name, which we fully control

Reply:
Yup, sure, I've just pushed a commit to use `finalizer/configmaps` instead.
Its using `/` instead of  `.`  like the other product finalizers as it would otherwise give an invalid finalizer error.

Looks to be due to this so could be different validation for finalizer between different resources :thinking: :
https://github.com/kubernetes/kubernetes/blob/86141c0cce2496508c2f9c1f1af3e6cbfb69f50c/pkg/apis/core/validation/validation.go#L5448

Reply:
@KevFan And have you tried adding it with our suffix, e.g. `finalizer.configmaps.integreatly.org` ?

Reply:
@matskiv Yup, got this error 
```
    Bootstrap stage reconcile failed: ConfigMap "cloud-resource-config" is
    invalid: metadata.finalizers[0]: Invalid value:
    "finalizer.configmaps.integreatly.org": name is neither a standard finalizer
    name nor is it fully qualified
```
I'm guessing it's due to missing a `/` in it's name from: 
https://github.com/kubernetes/kubernetes/blob/86141c0cce2496508c2f9c1f1af3e6cbfb69f50c/pkg/apis/core/validation/validation.go#L5446